solara/lab/components/confirmation_dialog.py

```diff
@@ -0,0 +1,49 @@
+from typing import cast, Callable, List, Union, Optional
+
+import solara
+import reacton.ipyvuetify as v
+
+
+@solara.component
+def ConfirmationDialog(
+    is_open_rv: solara.Reactive[bool],
+    action: Callable[[], None],
+    content: Union[str, solara.Element] = "",
+    title: str = "Confirm action",
+    on_close: Callable[[], None] = lambda: None,
+    ok: Union[str, solara.Element] = "OK",
+    cancel: Union[str, solara.Element] = "Cancel",
+    children: List[solara.Element] = [],
+):
+    def perform_action():
+        action()
+        close()
+
+    def close():
+        on_close()  # possible additional actions when closing
+        is_open_rv.set(False)
+
+    if is_open_rv.value:
+        with v.Dialog(
+            #children=children,
+            v_model=is_open_rv.value,
+            on_v_model=is_open_rv.set,
+            permanent=True,
+            max_width=500,
```

Comment:
@mariobuikhuizen any ideas on a good default width? If we do something like this (e.g. the default is not good enough) we should expose it as an argument.

Reply:
yeah, I think we should expose `max_width`, as one would want to tailor it to the content. I think 500px is a good default.